nx_parallel/algorithms/components/attracting.py

```diff
@@ -0,0 +1,35 @@
+from networkx.algorithms.components.attracting import attracting_components
```

Comment:
This can be `from networkx import attracting_components`. 
And that is the standard way to import within NetworkX. (It is a "flat" namespace to a large extent). There are a few places where it isn't flat: Many of the `bipartite` algorithms have the same names as the regular functions, so we put them into a sub-namespace: `nx.bipartite.closeness_centrality()`.  Of course people **can** call them with the full pathname. But we encourage people to use the shorter syntax.